eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -1,210 +1,164 @@
 # Spring Cloud Azure Stream Binder for Event Hub Code Sample shared library for Java
```

Comment:
The title of this article should be updated too.

Reply:
ok